locales/ru.yaml

```diff
@@ -120,7 +120,7 @@ sfc:
     tryAgain: ПОВТОРИТЬ ПОПЫТКУ
   SheetComboDialog:
     drawResults: Нарисовать Результаты
-    noDuplicate: Нету Дубликатов
-    # changeDrawSize: Change Draw Size
-    # allowDuplicate: Allow Duplicate
-    # blindfoldMode: Blindfold Mode
+    noDuplicate: Без Дубликатов
+    changeDrawSize: Изменить Размер Кисти
```

Comment:
The "draw" here refers to drawing something from a pool, not drawing something on paper.